ui/src/app/app.module.ts

```diff
@@ -22,7 +22,7 @@ import {environment} from 'environments/environment';
 
 /* Our Modules */
 import {AppRoutingModule} from 'app/app-routing.module';
-import {CohortSearchModule} from './cohort-search/cohort-search.module';
+import {CohortSearchModule} from './cohort-search/module';
```

Comment:
Should module name match the folder name?  In this case cohort-search(./cohort-search/module) doesn't match naming convention of (./cohort-review/cohort-review.module). This is an angular convention to name components/modules/etc.. the same as folder name. I'm not attached to this by an means but i like consistency.